ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -34,6 +37,42 @@ public static boolean shouldScrub(Map<String, String> attributes, JSONObject jso
             .filter(s -> s.contains("do not use eval with system privileges")).isPresent()) {
       countScrubbedBug1567596.inc();
       return true;
+    } else if ("telemetry".equals(attributes.get(ParseUri.DOCUMENT_NAMESPACE))
+        && "crash".equals(attributes.get(ParseUri.DOCUMENT_TYPE))
+        && (("nightly".equals(attributes.get(ParseUri.APP_UPDATE_CHANNEL))
+            && (attributes.get(ParseUri.APP_VERSION).startsWith("68")
+                || attributes.get(ParseUri.APP_VERSION).startsWith("69")))
+            || ("beta".equals(attributes.get(ParseUri.APP_UPDATE_CHANNEL))
+                && attributes.get(ParseUri.APP_VERSION).startsWith("68")))
+        && Optional.of(json) // payload.metadata.RemoteType
+            .map(j -> j.optJSONObject("payload")) //
+            .map(j -> j.optJSONObject("metadata")) //
+            .map(j -> j.optString("RemoteType")) //
+            .filter(s -> s.startsWith("webIsolated=")).isPresent()) {
+      // https://bugzilla.mozilla.org/show_bug.cgi?id=1562011
+      countScrubbedBug1562011.inc();
+      return true;
+    } else if ("telemetry".equals(attributes.get(ParseUri.DOCUMENT_NAMESPACE))
+        && "bhr".equals(attributes.get(ParseUri.DOCUMENT_TYPE))
+        && "nightly".equals(attributes.get(ParseUri.APP_UPDATE_CHANNEL))
```

Comment:
This match is less complicated than above because bhr is nightly-only? In which case, do we even care about filtering on nightly?

Reply:
Yep, it's nightly only. Good point.